lib/modules/@apostrophecms/ui/scss/_storybook.scss

```diff
@@ -0,0 +1,8 @@
+// Storybook specific override styles
```

Comment:
Should this be here? If this is storybook specific, couldn't it have a negative affect on the the UI?